index.html

```diff
@@ -591,8 +591,8 @@
 the victim.
 
 When an [=actor=] can collect [=data=] about a [=person=] and process it automatically, and that
-[=person=] cannot automatically protect their [=data=] and prevent its processing ([=automation asymmetry=])
-this creates an imbalance of power that favors that [=actor=] and decreases the [=person=]'s agency.
+[=person=] has to take manual action to protect their [=data=] and prevent its processing, this <dfn>automation asymmetry</dfn>
```

Comment:
Not about your particular change, but I noticed that this sentence implies that protecting and non-processing always go together, but I think user preferences aren't always that clear-cut. How about the following to imply that users need the option to make several different choices?

```suggestion
[=person=] has to take manual action to protect their [=data=] or control its processing, this <dfn>automation asymmetry</dfn>
```